src/org/traccar/database/NotificationManager.java

```diff
@@ -62,6 +66,9 @@ public void updateEvent(Event event, Position position) {
                 if (notification != null) {
                     if (notification.getWeb()) {
                         Context.getConnectionManager().updateEvent(userId, event);
+                        if (withPosition) {
+                            Context.getConnectionManager().updatePosition(userId, position);
```

Comment:
Please elaborate. It doesn't seem right to send position twice. Am I understanding correctly that it's what you are doing?